.github/CODEOWNERS

```
@@ -1 +1 @@
-* @zzstoatzz
\ No newline at end of file
+* @zzstoatzz @desertaxle
```

If we're updating the codeowners, we might as well set it to @PrefectHQ/integrations 
```suggestion
* @PrefectHQ/integrations 
```